os/board/rtl8730e/src/component/wifi/inic/inic_ipc_msg_queue.c

```diff
@@ -73,27 +73,18 @@ struct task_struct ipc_msgQ_wlan_task;
  */
 static sint enqueue_ipc_msg_node(struct ipc_msg_node *p_node, _queue *p_queue)
 {
-#if defined(CONFIG_SMP_NCPUS) && (CONFIG_SMP_NCPUS > 1)
```

Comment:
Please delete ipc_msg_queue_lock variable at line 61 as it is no longer used.

Reply:
Done.